scripts/ci/test_extensions.sh

```diff
@@ -19,7 +19,7 @@ output=$(az extension list-available --query [].name -otsv)
 exit_code=0
 
 # azure-cli-ml: https://github.com/Azure/azure-cli-extensions/issues/826
-ignore_list='azure-cli-ml fzf'
+ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s'
```

Comment:
- `arcappliance` is added by @pkoushik in https://github.com/Azure/azure-cli-extensions/pull/3079
- `arcdata` is added by @swells in https://github.com/Azure/azure-cli-extensions/pull/4069
- `connectedk8s` is added by @akashkeshari in https://github.com/Azure/azure-cli-extensions/pull/1570